protocol/indexer/events/update_clob_pair_create.go

```diff
@@ -0,0 +1,24 @@
+package events
+
+import (
+	v1 "github.com/dydxprotocol/v4-chain/protocol/indexer/protocol/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// NewUpdateClobPairEvent creates a UpdateClobPairEventV1
+// representing update of a clob pair.
+func NewUpdateClobPairEvent(
+	clobPairId uint32,
+	status types.ClobPair_Status,
+	quantumConversionExponent int32,
+	subticksPerTick uint32,
+	stepBaseQuantums uint64,
```

Comment:
Possible to use `satypes.BaseQuantums` and `types.SubticksPerTick`, and `types.ClobPairId` for the params?

Reply:
yeah sorry I just copied this from Teddy in the hard fork PR, will update